src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -191,4 +200,9 @@ public boolean isOverdue(ReadOnlyTask task) {
         return false;
     }
 
+  //@@author A0135998H
+    public boolean isFloating(ReadOnlyTask task) {
+        return !(task.getStartEndDateTime()).isPresent() && !(task.getDeadline()).isPresent();
```

Comment:
The thing about brackets is to help things become more understandable, in this case it looks as though you are negating a `ZonedDateTime` which is not possible so your brackets are wrong.

so it should be `return !(task.getStartEndDateTime().isPresent()) && !(task.getDeadline().isPresent());` if you want to keep the brackets